src/main/openapi/ds-image-openapi_v1.yaml

```diff
@@ -451,7 +459,10 @@ paths:
             content: 
               application/ld+json:
                 schema:
-                  $ref: '#/components/schemas/jsonld'
+                  $ref: '#/components/schemas/IIIF_info'
+              application/xml:
```

Comment:
`xml` har brug for et ydre element of i dette tilfælde for et namespace. Det kan angives under `IIIF_info` component med

```
      xml:
        name: info
        namespace: 'http://library.stanford.edu/iiif/image-api/ns/'
```

Reply:
Unfortunately it does not gets assigned as a default namespace. Bu we'll postpone this bug 'till later.